pyupgrade.py

```diff
@@ -917,10 +917,16 @@ def _simplify_conversion_flag(flag: str) -> str:
     return ''.join(parts)
 
 
+NAMED_UNICODE_RE = re.compile(r'(?<!\\)\\N{{([^}]+)}}')
```

Comment:
this one also doesn't work, I was hoping you'd copy the example I linked :) -- it needs to match (1) negative lookbehind for backslash (2) even-number-of-backslashes (3) \N